cli/azd/pkg/account/manager.go

```diff
@@ -2,82 +2,109 @@ package account
 
 import (
 	"context"
+	"errors"
 	"fmt"
-	"strings"
+	"log"
+	"os"
 
 	"github.com/azure/azure-dev/cli/azd/pkg/config"
 	"github.com/azure/azure-dev/cli/azd/pkg/tools/azcli"
 	"golang.org/x/exp/slices"
 )
 
+// JSON document path locations for default subscription & location
+const (
+	defaultSubscriptionKeyPath = "defaults.subscription"
+	defaultLocationKeyPath     = "defaults.location"
+)
+
+// The default location to use in AZD when not previously set to any value
+var defaultLocation Location = Location{
+	Name:        "eastus2",
+	DisplayName: "(US) East US 2",
+}
+
 // Manages azd account configuration
 // `az cli` is not required and will only be called `azd` default have not already been set.
 type Manager struct {
-	azCli azcli.AzCli
+	// Path to the local azd user configuration file
+	filePath      string
+	configManager config.Manager
+	config        config.Config
+	azCli         azcli.AzCli
 }
 
 // Creates a new Account Manager instance
-func NewManager(ctx context.Context) *Manager {
-	return &Manager{
-		azCli: azcli.GetAzCli(ctx),
+func NewManager(configManager config.Manager, azCli azcli.AzCli) (*Manager, error) {
+	filePath, err := config.GetUserConfigFilePath()
+	if err != nil {
+		return nil, err
 	}
+
+	azdConfig, err := configManager.Load(filePath)
+	if err != nil {
+		if errors.Is(err, os.ErrNotExist) {
+			log.Println(err.Error())
```

Comment:
Maybe something like log.Printf("configuration file %s does not exist, creating new empty config", filePath) or something else to let us note that the error we are logging is actually something we ignored?